README.md

```diff
@@ -0,0 +1,166 @@
+        <td>dual-core CPU</td>
+    </tr>
+    <tr>
+        <td>Memory (RAM)</td>
+        <td>4 GB</td>
+    </tr>
+    <tr>
+        <td>Graphics* (GPU)</td>
+        <td style="vertical-align:top">
+            Intel HD Graphics (Gen 7)<br/>
+            GeForce 8xxx series (or higher) or<br/>
+            Radeon HD 2000 series (or higher)<br/>
+            with OpenGL 3.3
+        </td>
+    </tr>
+    <tr>
+        <td>Storage (HDD)</td>
+        <td>1 GB</td>
+    </tr>
+</table>
+
+\* _Please note, that if you have both integrated (chip) and dedicated (card) graphics, you should make sure that you're actually using your dedicated graphics when running Terasology._
+
+Internet connectivity is required for downloading Terasology via the Launcher, afterwards playing offline is possible.
+
+For easy game setup (recommended) you can use our launcher - [download it here](https://terasology.org/download.html).
+
+For more information about playing, like hot keys or server hosting, see the [dedicated page](docs/Playing.md) or check out the [modules](docs/Modules.md).
+
+
+### Alternative Installation Methods
+
+If you already have a Java Development Kit (JDK) installed, you may use a direct download release as an alternative to using the [launcher](https://github.com/MovingBlocks/TerasologyLauncher/releases). Java version 11 is required.
+
+Direct download stable builds are uploaded to [our release section here on GitHub](https://github.com/MovingBlocks/Terasology/releases) while the cutting-edge develop version can be downloaded direct [here from our Jenkins](https://jenkins.terasology.io/teraorg/job/Terasology/job/Omega/job/master/lastSuccessfulBuild/artifact/distros/omega/build/distributions/TerasologyOmega.zip).
+
+
+## Development
+
+Development is possible on all common platforms (Windows, Linux, MacOS) as long as the JDK is properly set up.
+
+### Requirements
+
+Technical Requirements:
+- Java SE Development Kit (JDK) 11. It may be possible to use a later version of the JDK for local development, but the CI will verify against this baseline version.
+- Git to clone the repo and commit changes.
+
+Non-Technical Requirements:
+- familiarity with Git. Have a look at https://learngitbranching.js.org/ if you're not familiar with Git yet.
+- familiarity with GitHub, _especially forks_. Have a look at [GitHub's "Working with Forks" Guide](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks) if you don't know how to work with forks yet.
+
+### Workspace Setup
+
+To be able to run **Terasology** from source, you'll need to setup your workspace.
+Follow the [Contributor Quick Start Guide](https://github.com/MovingBlocks/Terasology/wiki/Contributor-Quick-Start).
+This guide is designed for [IntelliJ IDEA](http://www.jetbrains.com/idea/) (you can use the free community edition), but alternative setups are possible.
+
+:warning: _Note, that a Terasology workspace is a **multi-repo workspace**._
```

Comment:
To highlight these callouts even more, I like to make them quotes:

```suggestion
> :warning: _Note, that a Terasology workspace is a **multi-repo workspace**._
```

Reply:
done